2020-03-15 18:39:10,444 [ERROR   ]: E MAIN Log level: 0                                                                                                                                    [main.py:main:183]
2020-03-15 18:39:10,444 [WARNING ]: W Adding 2+3=5                                                                                                                                         [main.py:add:130]
2020-03-15 18:39:10,445 [DEBUG   ]: D MAIN add_result: 5                                                                                                                                   [main.py:main:185]
2020-03-15 18:39:10,445 [DEBUG   ]: D UTILS Multiplying 2*3=6                                                                                                                              [utils.py:multiply:6]
2020-03-15 18:39:10,445 [INFO    ]: I UTILS Multiplying 2*3=6                                                                                                                              [utils.py:multiply:7]
2020-03-15 18:39:10,446 [WARNING ]: W UTILS Multiplying 2*3=6                                                                                                                              [utils.py:multiply:8]
2020-03-15 18:39:10,446 [ERROR   ]: E UTILS Multiplying 2*3=6                                                                                                                              [utils.py:multiply:9]
2020-03-15 18:39:10,446 [CRITICAL]: C UTILS Multiplying 2*3=6                                                                                                                              [utils.py:multiply:10]
2020-03-15 18:39:10,447 [INFO    ]: I MAIN multiply_result: 6                                                                                                                              [main.py:main:187]
